src/pages/favoritesPage/favouritesPage.tsx

```diff
@@ -0,0 +1,39 @@
+import style from './favoritesPage.module.scss';
+import { useContextProvider } from '../../context/ProductsContext';
+import { ProductCard } from '../../components/Product_card/Product_card';
+
+export const FavouritesPage = () => {
+  const { favourites } = useContextProvider();
+  const favouritesCount = favourites.length;
+  const isPlural = favouritesCount !== 1;
```

Comment:
favouritesCount > 1

Reply:
you add 's' if there is more than one product, it would be more correct to write
favoritesCount > 1